packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -205,6 +205,32 @@ class WebViewController {
     return _channel.invokeMethod('loadUrl', url);
   }
 
+  /// Check that there is back item in the history list.
```

Comment:
Suggestions: `Checks whether there's a back history item.`.

Reply:
It looks good! -> 1df519b